src/server.ts

```diff
@@ -230,6 +231,9 @@ connection.onDidChangeConfiguration((change) => {
 			if (settings.yaml.format.bracketSpacing === false) {
 				yamlFormatterSettings.bracketSpacing = false;
 			}
+			if (settings.yaml.format.enable) {
+				yamlFormatterSettings.enable = settings.yaml.format.enable || true;

```

Comment:
Maybe I'm misunderstanding something here, but will this ever set `yamlFormatterSettings.enable` to `false`?

If `settings.yaml.format.enable` is `false`, won't this code block just be ignored?

Reply:
And won't the `|| true` ensure that this would always be set to `true` anyway?

But I'm bad at Javascript so maybe this actually works.

Reply:
And at Typescript :)

Reply:
Yep! Good catch! I must not have had my coffee yet when I made the PR